phpcs.xml.dist

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<ruleset name="Genesis Sample Code Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki -->
+	<!-- See https://github.com/wimg/PHPCompatibility -->
+	<!-- See https://github.com/Automattic/_s/blob/master/phpcs.xml.dist -->
+
+	<description>Custom rules for Genesis Sample.</description>
+
+	<!-- Check files in this directory and subdirectories. -->
+	<file>.</file>
+
+	<!-- Prevent sniffs of some directories. -->
+	<exclude-pattern>/vendor/</exclude-pattern>
+
+	<!-- Pass flags to PHPCS:
+		 p: Show progress of the run.
+		 s: Show sniff codes in all reports.
+		 v: Print verbose output.
+	-->
+	<arg value="psv"/>
+
+	<!-- Only check PHP and CSS files. -->
+	<arg name="extensions" value="php,css"/>
+
+	<!-- Use the WordPress ruleset, with exclusions. -->
+	<rule ref="WordPress">
+		<exclude name="WordPress.VIP"/>
+		<exclude name="Squiz.PHP.CommentedOutCode.Found" />
```

Comment:
There's an inconsistent mix of spaces before the `/>` in this file.

Reply:
Nice catch, thanks.